### PR TITLE
[WIP] Build new pages for coronavirus accordion edit and show

### PR DESCRIPTION
**Pull request by DilwoarH**
## What
Build the frontend for:
- overview page
- editing a section

## Why
These screens are the main interface for the MVP publishing app.

https://trello.com/c/pXFRbJXK/357-story-1b-build-new-pages-for-coronavirus-accordion-edit-and-show

### CODE REVIEW

File: app/views/coronavirus/edit_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "Status" => "Draft",
+    "Last saved" => format_full_date_and_time(DateTime.now),
+  }
+%>
+
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, "Coronavirus (COVID-19)" %>
+<% content_for :context, 'Edit guidance and support accordion' %>
+
+<div class="covid-manage-page govuk-grid-row">
```
Comment (hannako) [inline app/views/coronavirus/edit_page.html.erb:22]: Not sure if it's just me but looks like the alignment is a bit off here?

Comment (danacotoran) [reply app/views/coronavirus/edit_page.html.erb]: Yep you are correct – this should have been resolved now

**Comment by danacotoran** [comment]
> Looking good! I checked out out the branch to review. So that we can merge please could you remove the routes, and could you move the files:
> manage_page --> views/coronavirus/publishing/show.html.erb
> edit_page --> views/coronavirus/publishing/edit.html.erb
> and move the partials into the publishing folder as well.

Thanks for the comment @hannako  🙌  I've updated this PR based on your notes, would you mind checking that all looks as intended now?

File: app/controllers/coronavirus_controller.rb [hunk 2]
```diff
@@ -37,6 +37,14 @@ def publish
     redirect_to coronavirus_path(slug)
   end
 
+  def manage_page
```
Comment (hannako) [inline app/controllers/coronavirus_controller.rb:40]: Would you mind just deleting these two actions then feel free to merge!

Comment (danacotoran) [reply app/controllers/coronavirus_controller.rb]: thank you @hannako, will be merging as soon as the checks are finished  🚀

**Pull request merged**